sphinx/sphinx.gradle

```diff
@@ -8,12 +8,22 @@ description = 'Generates docs for the python libraries provided in Deephaven Cor
 
 evaluationDependsOn ':Integrations'
 
+tasks.register('copyPyClientWhl', Copy) {
```

Comment:
unless anything else is writing to build/wheel, prefer Sync to Copy, so that when version numbers change we don't require a `./gradlew clean` to mop up old files